boards/arduino-mega2560/include/atmega_pcint.h

```diff
@@ -0,0 +1,16 @@
+#ifndef ATMEGA_PCINT_H
+#define ATMEGA_PCINT_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define ATMEGA_PCINT_MAP_PCINT0 GPIO_PIN(PORT_B, 0), GPIO_PIN(PORT_B, 1), GPIO_PIN(PORT_B, 2), GPIO_PIN(PORT_B, 3), GPIO_PIN(PORT_B, 4), GPIO_PIN(PORT_B, 5), GPIO_PIN(PORT_B, 6), GPIO_PIN(PORT_B, 7)
+#define ATMEGA_PCINT_MAP_PCINT1 GPIO_PIN(PORT_E, 0), GPIO_PIN(PORT_J, 0), GPIO_PIN(PORT_J, 1), GPIO_UNDEF, GPIO_UNDEF, GPIO_UNDEF, GPIO_UNDEF, GPIO_UNDEF
+#define ATMEGA_PCINT_MAP_PCINT2 GPIO_PIN(PORT_K, 0), GPIO_PIN(PORT_K, 1), GPIO_PIN(PORT_K, 2), GPIO_PIN(PORT_K, 3), GPIO_PIN(PORT_K, 4), GPIO_PIN(PORT_K, 5), GPIO_PIN(PORT_K, 6), GPIO_PIN(PORT_K, 7)
+
+#ifdef __cplusplus
+}
+#endif
+
+#endif /* ATMEGA_PCINT_H */
```

Comment:
This file now conflicts with the test: The test sets up also PCINT interrupts for pins not connected to the Arduino Mega2560 header. (If I recall correctly it was my suggestion to override the pcint mapping for this board. But now I think this was a bad idea. E.g. other RIOT code does not prevent the user from using pins available at the MCU but not connected to pin header. Maybe it would be better to remove this file and stick with the default mapping for the ATmega2560 MCU?)

Reply:
What is the actual conflict here? If the pins are not correct for that board, I think it's totally fine to override the pcint definition. The test should then not setup pcints that are not available. Or better: the test should expect the gpio_init_int() call to fail, if the pin is invalid on the board.

Reply:
> Or better: the test should expect the gpio_init_int() call to fail, if the pin is invalid on the board. 

+1